### ISSUE TITLE
The window closing when trying to select file in hidden folder

### ISSUE THREAD

**Issue by darknate** [codebasesaga/Workbench#33]
Trying to pick the folder on /Library/blabla/, but the finder windows close when picking the Library folder: https://take.ms/pUyHH
Catalina 10.15

**Comment by jasikpark** [comment]
I am having a similar problem - I switched over to zsh, so I'm trying to add `.zshrc` to my Workbench app, and the file picker keeps closing before actually selecting the file.

https://imgur.com/a/LhmTOS5

![ezgif com-optimize](https://user-images.githubusercontent.com/10626596/71290701-1d991800-2336-11ea-980b-611547630180.gif)

**Comment by marcoacierno** [comment]
Have you found a workaround? 
EDIT: I was able to add files using the keyboard instead of the mouse

**Comment by limex** [comment]
Any other idea how to fix this? 
I understand that Max is busy doing other amazing stuff on top of brew ... Anybody able to rebuild the code for the newer versions of macos?

**Comment by mxcl** [comment]
yeah I am sorry about this bug, I never managed to figure out what was happening that caused it despite looking quite long and hard.

I believe I found a workaround, though I myself have stopped using this.

**Comment by limex** [comment]
>  I myself have stopped using this.

ah. ok. Do you have other best practices for backup/sync of .dotfiles without using the Workbench app?

**Comment by mxcl** [comment]
I have instead gone back to manual sync :/

* https://fig.io recently added this feature
* tea/gui will support this (not out yet)
